Components/LandingNav.tsx

```diff
@@ -45,51 +48,73 @@ const LandingNav = () => {
-          <Link className={navLinkStyle} href="#ourservices" onClick={handleOurServicesClick}>
-            Our Services
-          </Link>
-          <Link className={navLinkStyle} href="#benefits" onClick={handleBenefitsClick}>
-            Benefits
-          </Link>
-          <Link className={navLinkStyle} href="/faq">
-            {/* adding badge on div for new */}
-            <div>
-              FAQ
-              <span className="absolute px-1 py-1 ml-2 text-xs text-white bg-red-500 rounded-full -top-2">
-                {/* on hover show popup - attention needed */}
-                <div className="w-2 h-2 shadow-xl shadow-red-100"></div>
-
-              </span>
-
-            </div>
-          </Link>
-        </div>
+        {!pageNotFound && (
+          <div className="flex flex-row items-center gap-8">
+            <Link className={navLinkStyle} href="#">
+              Home
+            </Link>
+            <Link
+              className={navLinkStyle}
+              href="#"
+              onClick={handleCoursesClick}
+            >
+              Courses
+            </Link>
+            <Link
+              className={navLinkStyle}
+              href="#ourservices"
+              onClick={handleOurServicesClick}
+            >
+              Our Services
+            </Link>
+            <Link
+              className={navLinkStyle}
+              href="#benefits"
+              onClick={handleBenefitsClick}
+            >
+              Benefits
+            </Link>
+            <Link className={navLinkStyle} href="/faq">
+              {/* adding badge on div for new */}
+              <div>
+                FAQ
+                <span className="absolute px-1 py-1 ml-2 text-xs text-white bg-red-500 rounded-full -top-2">
+                  {/* on hover show popup - attention needed */}
+                  <div className="w-2 h-2 shadow-xl shadow-red-100"></div>
+                </span>
+              </div>
+            </Link>
+          </div>
+        )}
         {/* Get Started Button and Dark Mode Button*/}
         <div className="flex flex-row gap-4 items-center">
-        <div
-          onClick={handleGetStartedClick}
-          className="flex flex-row items-center gap-4 p-1 border-2 border-black rounded-full cursor-pointer dark:border-zinc-50"
-        >
-          <h4 className="ml-2 font-medium ">Get Started</h4>
-          <span className="p-2 text-white bg-black dark:text-zinc-800 dark:bg-zinc-50  rounded-full material-icons">
-            arrow_outward
-          </span>
-        </div>
-        <div className="mt-1">
-          <DarkModeToggle />
-        </div>
+          <div
+            onClick={!pageNotFound ? handleGetStartedClick : handleGoBack}
+            className="flex flex-row items-center gap-4 p-1 border-2 border-black rounded-full cursor-pointer dark:border-zinc-50"
+          >
+            <h4 className="ml-2 font-medium ">
+              {!pageNotFound ? `Get Started` : `Go Back`}
```

Comment:
Instead of showing Go back button in navbar, please show this button below the text - `Sorry, there is nothing to see here!`

Reply:
Done